.docs/user-guide/storage-providers.md

```diff
@@ -25,14 +25,14 @@ Provider              | Storage Platform(s)
 ----------------------|--------------------
 EMC | [ScaleIO](http://libstorage.readthedocs.io/en/stable/user-guide/storage-providers/#scaleio), [Isilon](http://libstorage.readthedocs.io/en/stable/user-guide/storage-providers/#isilon)
 [Oracle VirtualBox](http://libstorage.readthedocs.io/en/stable/user-guide/storage-providers/#virtualbox) | Virtual Media
+Amazon EC2 | [EBS](http://libstorage.readthedocs.io/en/stable/user-guide/storage-providers#aws-ebs), [EFS](http://libstorage.readthedocs.io/en/stable/user-guide/storage-providers#aws-efs)
```

Comment:
Nevermind, I see what is actually occurring here. I was wrong. Thanks @codenrhoden!